pom.xml

```diff
@@ -558,6 +558,12 @@
                 <artifactId>sentry</artifactId>
                 <version>${sentry.version}</version><!--$NO-MVN-MAN-VER$-->
             </dependency>
+            <dependency>
```

Comment:
Why we need this now?

Reply:
h2 is pulled as a transitive dependency. I added this to override the bo version